### DISCUSSION TITLE
What is the correct / recommended way to create such layout?

### DISCUSSION THREAD

**Discussion by cyfung1031** [quasarframework/quasar#10704] [General - Components / Directives / etc] [answered]
![image](https://user-images.githubusercontent.com/44498510/133684808-3f2ea1b5-7207-4406-ac28-6575c9c1dc47.png)

In the official docs, the scrollbar is to the overall page.

I tried to make it like
```
<div class="flex-row">
    <div>.....header....</div>
    <div style="flex: 1 1 auto;">
          <q-layout container  >
                <teleport to="body" :disabled="$q.platform.is.desktop">
                    <q-drawer mini-to-overlay :mini="miniState" bordered v-model="drawer" @mouseenter="miniState = false" @mouseleave="miniState = true"> .... </q-drawer>
                </teleport>
                <q-page-container>
                    <q-page class="flex-row">
                        <div style="flex: 1 1 auto;"> .... content ... </div>
                    </q-page>
                </q-page-container>
          </q-layout >
    </div>
</div>
```

It can work on both mobile and dekstop, but I cannot disable the scrolling and fix the full height of the content area.
Any correct / recommended way to create such layout?

**Comment by metalsadman** [comment]
you can use [QScrollArea](https://quasar.dev/vue-components/scroll-area) for the green part.

**Reply by cyfung1031** [reply]
Great. I see what you means. Coz I cannot find any example from the official docs for qlayout combining with QScrollArea.

How about `<q-page-scroller>`? I see it can be used in `<q-page-container>`, but is there similar thing for QScrollArea??

**Reply by cyfung1031** [reply]
#10715  I writed coding for such features. Somehow the objective is achieved. Thanks.

**Comment by cyfung1031** [comment, marked as answer]
Full Coding for the implemenation: https://codepen.io/cyfung1031/pen/wveyPwv

I have also created an issue to ask for adding this to the official docs as an example to use QLayout with QScrollArea #10720
